main_test.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"os"
+	"testing"
+)
+
+func TestScrape(t *testing.T) {
+
+	// TestSuccessfulConnection
+	exporter, _ := NewExporter("credentials.json", os.Getenv("GCP_PROJECT"))
```

Comment:
What project are you planning to use for this test ?

Reply:
I wanted to ask you guys before I added this to CI. Locally I've just been using our dev project & credentials.